kong/dao/schemas_validation.lua

```diff
@@ -117,9 +117,16 @@ function _M.validate_entity(tbl, schema, options)
               is_valid_type = bool == "true" or bool == "false"
               t[column] = bool == "true"
             elseif v.type == "array" then
-              t[column] = utils.strip(t[column]) == "" and {} or utils.split(t[column], ",") -- Handling empty arrays
+              -- handle escaped commas inside the comma-separated array
+              -- by flipping them into a \0 and then back
+              local escaped = t[column]:gsub("\\,", "\0")
+              if utils.strip(t[column]) == "" then
+                t[column] = {} -- Handling empty arrays
+              else
```

Comment:
Got it. I tend to use line-breaks as sort of "paragraph" breaks when separating logical parts of a function, so for larger blocks they make sense but in a block like that where it would be the only line break, my reaction looking at the resulting code with the break was that it looked weird, because it draws attention to the "else" more than anything in that block. But this revision is outdated by now :)